driver/bpf/filler_helpers.h

```diff
@@ -994,6 +1047,51 @@ static __always_inline int __bpf_val_to_ring(struct filler_data *data,
 	return PPM_SUCCESS;
 }
 
+static __always_inline enum read_memory param_type_to_mem(enum ppm_param_type type)
+{
+	/* __bpf_val_to_ring() uses bpf_probe_read_* functions for particular types
+	 * only.  Instead of changing all places, let's keep it simple and try to
+	 * spot the correct address space by type.
+	 */
+
+	switch (type)
+	{
+	case PT_CHARBUF:
+	case PT_FSPATH:
+	case PT_FSRELPATH:
+	case PT_BYTEBUF:
+	{
+		/* Those above require some review on whether USER reads are
+		 * correct. If not, explicit use the respective helper with
+		 * the _mem() suffix to specify the memory to read from.
+		 *
+		 * See also the usage below in the helpers.
+		 */
+		return USER;
+	}
+	default:
+	{
+		return KERNEL;
+	}
```

Comment:
[nit] I don't think you need braces here
```suggestion
		/* Those above require some review on whether USER reads are
		 * correct. If not, explicit use the respective helper with
		 * the _mem() suffix to specify the memory to read from.
		 *
		 * See also the usage below in the helpers.
		 */
		return USER;

	default:
		return KERNEL;
```

Reply:
Done. Thank you!